gulpfile.js

```diff
@@ -0,0 +1,46 @@
+(function () {
```

Comment:
More of a larger question outside of this PR... Should we break up this gulpfile to mirror what [we've done in the UXPL](https://github.com/edx/ux-pattern-library/tree/master/gulp) (and @AlasdairSwan and I tried to do with https://github.com/edx/edx-platform/pull/7175)? There has been some maintenance and abstraction benefits for us in maintaining the doc site and all of our Sass, images, templating tasks.